pipeline/leech/main.go

```diff
@@ -44,6 +44,7 @@ func init() {
 	register.DoFn2x1[context.Context, leech.EventRecord, leech.LeechRecord](&leech.IngestLogsFn{})
 }
 
+// why is this file here and not in cmd/? is it because its not intended to be run from cli?
```

Comment:
Apache Beam pipelines aren't really executable in the same way as the rest of our commands. Putting it in `cmd` comes with an expectation that it is runnable without a beam runner as is which it is not.